core/templates/dev/head/pages/library/SearchBarDirective.js

```diff
@@ -25,17 +25,17 @@ oppia.directive('searchBar', [
         'search_bar_directive.html'),
       controller: [
         '$scope', '$rootScope', '$timeout', '$window', '$location',
```

Comment:
If we are sorting everything then lets sort this line too:
`'$location', '$rootScope', '$scope',  '$timeout', '$window', `

Reply:
Done.